gc/base/MemorySubSpaceUniSpace.cpp

```diff
@@ -724,4 +728,43 @@ MM_MemorySubSpaceUniSpace::adjustExpansionWithinSoftMax(MM_EnvironmentBase *env,
 		}
 	}
 	return expandSize;
-}	
+}
+
+uintptr_t
+MM_MemorySubSpaceUniSpace::getHeapFreeMaximumHeuristicMultiplier(MM_EnvironmentBase *env)
+{
+	uint32_t gcPercentage;
+	uint32_t expectedGcPercentage;
+	uint32_t gcRatio;
+
+	if (NULL != _collector) {
+		gcPercentage = _collector->getGCTimePercentage(env);
+	} else {
+		gcPercentage = _extensions->getGlobalCollector()->getGCTimePercentage(env);
+	}
+
+	expectedGcPercentage = (_extensions->heapContractionGCTimeThreshold + _extensions->heapExpansionGCTimeThreshold) / 2;
+	gcRatio = gcPercentage / expectedGcPercentage;
+
```

Comment:
Introduce a new trace point that will report the returned value. Same for the other (min) helper.